modules/rhel-compute-updating.adoc

```diff
@@ -90,15 +86,15 @@ NAME                        STATUS                        ROLES    AGE    VERSIO
 mycluster-control-plane-0   Ready                         master   145m   v1.23.0
```

Comment:
This section `Review the current node status to determine which {op-system-base} worker to update` is not required , we don't need to review which RHEL worker could be updated in advance, we can add them to the inventory directly.